packages/core-contracts/contracts/mocks/Destroyer.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
Should we get rid of this one until we need it back?

I.e. if its not used in the current state, it should be deleted IMO.